README.md

```diff
@@ -22,23 +22,121 @@ At this point you can either run the repo using Docker or directly with Ruby.
 
 Using Docker, run the following command:
 
-* `docker-compose up`
+- `docker-compose up`
 
 This starts up a web server at <http://localhost:4000>, builds the static site and allows you to view changes live.
 
 _This has been tested with Docker version 2.3.0.3. See [Docker installation instructions](https://docs.docker.com/get-docker/)._
 
 _Note: On Windows you will need to re-run the `docker-compose up` command to see changes, unless you are using [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10)._
 
+## Publishing to JSBin
+
+This configuration is not required for basic documentation creation and modification, so most
+editors can simply skip to [Forking and running locally](#forking-and-running-locally).
+
+If you will be creating or modifying code within this repository that is to be uploaded automatically
+to [JSBin](https://jsbin.ably.io/), for our "Try it now code editor", then you will need to create yourself a JSBin config file:
+
+### Ably staff members
```

Comment:
I don't see a section for non-staff members. While not a priority to get this working for non-staff members we shouldn't leave open ended headings like this.

Reply:
Non-staff can't update JSBins

Reply:
(I'll delete that heading)